lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -1187,3 +1199,29 @@ def _get_environment_information(self, environment: Environment) -> Dict[str, st
                     node.log.exception(f"error on get {key}.", exc_info=identifier)
 
         return information
+
+    def _enable_libvirt_debug_log(self) -> None:
+        sed = self.host_node.tools[Sed]
+        sed.append(
+            f'log_outputs="1:file:{self.LIBVIRT_DEBUG_LOG_PATH} 3:syslog:libvirtd"',
+            "/etc/libvirt/libvirtd.conf",
+            sudo=True,
+        )
+
+        self.host_node.tools[Service].restart_service("libvirtd")
+
+    def _capture_libvirt_logs(self) -> None:
+        libvirt_log_path = self.host_node.local_log_path / "libvirtd.log"
+
+        if self.platform_runbook.capture_libvirt_debug_logs:
+            user = self.host_node.tools[Whoami].get_username()
+            self.host_node.tools[Chown].change_owner(self.LIBVIRT_DEBUG_LOG_PATH, user)
```

Comment:
This seems like it might cause problems, especially if run on someone’s dev box. I recommend copying the file to `working_dir` first, changing the permissions on that file and then copying to the test runner machine.

Reply:
What kind of problems do you expect this might cause?

Reply:
You shouldn’t change permissions in system directories to be that of a user.

Reply:
Done.